short/deploy-to-azure-from-github/README.md

```diff
@@ -0,0 +1,199 @@
+                                
+                        </ul>
+                        <h3>Job Title 2</h3>
+                        <p>
+                            Organization Name Goes Here | Start Month – End Month
+                        </p>
+                        <ul>
+                                <li>Task 1 - Write what you did!</li>
+                                <li>Task 2 - Write what you did!</li>
+                                <li>Write the outcomes/impact of your contribution</li>
+                                
+                        </ul>
+                    </section>
+                </article>
+            </main>
+        </body>
+    </html>
+```
+
+
+3. Open the style.css file and copy in the code below.
+ 
+```
+body {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 16px;
+    max-width: 960px;
+    margin: auto;
+}
+h1 {
+    font-size: 3em;
+    letter-spacing: .6em;
+    padding-top: 1em;
+    padding-bottom: 1em;
+}
+
+h2 {
+    font-size: 1.5em;
+    padding-bottom: 1em;
+}
+
+h3 {
+    font-size: 1em;
+    padding-bottom: 1em;
+}
+main { 
+    display: grid;
+    grid-template-columns: 40% 60%;
+    margin-top: 3em;
+}
+header {
+    text-align: center;
+    margin: auto 2em;
+}
+
+section {
+    margin: auto 1em 4em 2em;
+}
+
+i {
+    margin-right: .5em;
+}
+
+p {
+    margin: .2em auto
+}
+
+hr {
+    border: none;
+    background-color: lightgray;
+    height: 1px;
+}
+
+h1, h2, h3 {
+    font-weight: 100;
+    margin-bottom: 0;
+}
+#mainLeft {
+    border-right: 1px solid lightgray;
+}
+```
+4. **[Optional]** Add a `codeswing.json` file and paste in the code below. Then install the Codeswing extension on vscode.dev  
+```
+{
+"scripts": [],
+"styles": []
+}
+```
+5. Log in to [Azure Portal](portal.azure.com) and create a Static Web App
+
+    a) _Provide static web app details_
+
+    b) _Connect to your GitHub project from Azure_
+    
+    c) _Provide deployment details to configure your Continous Deployment (CD) GitHub Action workflow file_ and **Create**
+![create a static web app demo](./img/create-static-web-app-full.gif)
+
+6. Once your static web app is created successfully, Click 'Go to Resource' and then browse
+7. Head back to your project on github and observe under the 'GitHub Actions' tab. After the action runs succesfully and you get a green check mark, navigate back to the site and refresh
+8. Congratulations!! You have created your resume website and deployed it to a Static Web App
+
+## Next Steps
+Finish creating your resume website and deploy it on Azure for free. Share a link to your resume site on social media and tag us 😉:
+- @juliamuiruri4 (Twitter) & @juliamuiruri (LinkedIn)
+- #MSFTStudentSummit
+
+See [full workshop](https://github.com/microsoft/workshop-library/blob/main/full/build-resume-website/README.md/?WT.mc_id=academic-70942-juliamuiruri) to learn more
```

Comment:
GitHub URLs don't work with tracking links. And this link is using a code that isn't used in the rest of the file. Consider using the code everywhere else or not at all if it doesn't apply

Reply:
Thanks for pointing this out